plugin/mutate/source/dextool/plugin/mutate/backend/database/standalone.d

```diff
@@ -77,6 +77,35 @@ struct Database {
         return res.oneValue!long != 0;
     }
 
+    bool isInMutationStatusTable(MutationStatusId status_id) {
```

Comment:
*Note*: avoid leaking implementation details via the API.
*Note*: YAGNI.
I think you should merge isInMutationStatusTable and lostMarking. isInMutationStatusTable is how it is implemented in the database, lostMarking is the external functionality.
Or if you feel that it has wider use in the future I would recommend changing the name to what is externally expected such as "bool exists(MutationStatusId id)".

Reply:
Very true. I could see a use in the future where it is used, so I'll go with the namechange (plus merge that is)